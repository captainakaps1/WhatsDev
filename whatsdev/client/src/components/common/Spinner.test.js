import Spinner from "./Spinner";
import { render } from "@testing-library/react";

describe("Component Test - InputGroup", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment()).toMatchSnapshot();
  });
});
