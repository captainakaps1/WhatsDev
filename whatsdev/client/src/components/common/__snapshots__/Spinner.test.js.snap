// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Test - InputGroup matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="Loading..."
      src="loading.gif"
      style="width: 100px; margin: auto; display: block;"
    />
  </div>
</DocumentFragment>
`;
